build/assets/custom-dictionary.txt

```diff
@@ -418,8 +417,41 @@ SVs
 CircleCI
 ManuBot
 reproducibility
+oncoprint
+MAPK
+RAS
+upregulation
+KRAS
+LGATs
+ATRTs
+ETMRs
+rhabdoid
+teratoid
+oligodendroglial
+DMGs
+oligodendroglioma
+downregulation
+craniopharayngiomas
+desmoplastic
+dysembryoplastic
+barplot
+ependymomal
+fibrillary
+ganglioglioma
+ganglioneuroblastoma
+glial
+GNT
+melanocytic
+meningiomas
+meningothelial
+mesenchymal
+paraspinal
+pleomorphic
+sellar
+Subependymal
```

Comment:
Probably don't want this capitalized in the dictionary, or it will be treated as a proper noun...
```suggestion
subependymal
```